tests/integration/multi_index_tests.cpp

```diff
@@ -18,7 +18,7 @@ using namespace eosio::testing;
 
 BOOST_AUTO_TEST_SUITE(multi_index_tests)
 
-// this test is copy from leap test_api_multi_index
+// this test is copy from spring test_api_multi_index
```

Comment:
The `s` in this `spring` should be capitalized as `Spring`.

Reply:
Why?